ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-ospf.j2

```diff
@@ -29,6 +29,35 @@ router ospf {{ process_id }}
 {%     if router_ospf.process_ids[process_id].max_lsa is arista.avd.defined %}
    max-lsa {{ router_ospf.process_ids[process_id].max_lsa }}
 {%     endif %}
+{%     if router_ospf.process_ids[process_id].timers.lsa.rx is arista.avd.defined %}
+   timers lsa rx min interval {{ router_ospf.process_ids[process_id].timers.lsa.rx }}
+{%     endif %}
+{%     if router_ospf.process_ids[process_id].timers.lsa.tx.delay is arista.avd.defined %}
```

Comment:
this should check for all the child values at once, so we don't generate config with only some of the values. same for spf.

Reply:
Fixed with commit e4265b9a